app/models/curator/filestreams/audio.rb

```diff
@@ -9,5 +9,39 @@ class Filestreams::Audio < Filestreams::FileSet
     has_one_attached :document_access
     has_one_attached :document_master
     has_one_attached :text_plain
+
+    def derivatives_complete?
```

Comment:
Couple of thoughts here.

1. Should we change this method name to `required_derivatives_complete?` I'm thinking of a future scenario where we want to have a method that returns whether _any_ derivatives (including those that are not explicitly required, like an `:image_thumbnail_300` for `Filestreams::Metadata` for example) are complete.

2. I wonder whether it might be slightly DRY-er to have a method in each filestream class called `required_derivatives`, which simply returns an array of attachment names. Then we could move the `derivatives_complete?` method to `Filestreams::FileSet`, which would use the `required_derivatives` output to eval completeness, like:
```ruby
# just winging it here
def derivatives_complete?
  required_derivatives.all? { |a| a.attached? }
end
```

Also, can we create some specs for this method?